ts/screens/preferences/PreferencesScreen.tsx

```diff
@@ -88,34 +88,43 @@ class PreferencesScreen extends React.Component<Props> {
         subtitle={I18n.t("preferences.subtitle")}
         contextualHelp={contextualHelp}
       >
-        <Content>
+        <Content noPadded={true}>
```

Comment:
is this line the only thing that needs to be changed for this fix? everything else it seems refactoring the existing code?

Reply:
No, I also had to move the `<ListItem>` out from the `<PreferenceItem>` to fix the style.